### PR TITLE
Add Module connection strings in Jenkins

### PR DESCRIPTION
**Pull request by jenoola**
Fixes issue 10 for Azure-Samples/azure-iot-samples-csharp [https://github.com/Azure-Samples/azure-iot-samples-csharp/issues/10](url)

### CODE REVIEW

**Comment by CIPop** [comment]
Blocked due to internal CI issues.

**Comment by jenoola** [comment]
@CIPop reset to template changes only

**Comment by CIPop** [comment]
@jenoola please resolve the conflict before we can merge.

File: e2e/test/prerequisites/iot_config.cmd_template [hunk 1]
```diff
@@ -14,6 +14,13 @@ set IOTHUB_CONN_STRING_CSHARP=
 set IOTHUB_DEVICE_CONN_STRING=
 set IOTHUB_DEVICE_CONN_STRING2=
 
+@REM IoT Hub Module connection string:
+set IOTHUB_MODULE_CONN_STRING=
+set IOTHUB_MODULE_CONN_STRING2=
+
+@REM EventHub consumer group:
+set IOTHUB_EVENTHUB_CONSUMER_GROUP=e2e-tests
```
Comment (CIPop) [inline e2e/test/prerequisites/iot_config.cmd_template]: This is incorrect - it's already added below. Also, please don't add values in this file - we'd like it generic.

File: e2e/test/prerequisites/iot_config.sh_template [hunk 2]
```diff
@@ -15,6 +15,13 @@ export IOTHUB_CONN_STRING_CSHARP=""
 export IOTHUB_DEVICE_CONN_STRING=""
 export IOTHUB_DEVICE_CONN_STRING2=""
 
+# IoT Hub Module connection string:
+export IOTHUB_MODULE_CONN_STRING=""
+export IOTHUB_MODULE_CONN_STRING2=""
+
+# EventHub consumer group:
+export IOTHUB_EVENTHUB_CONSUMER_GROUP=e2e-tests
```
Comment (CIPop) [inline e2e/test/prerequisites/iot_config.sh_template]: Same as above - already existing in the file.

**Comment by CIPop** [comment]
Due to CI intermittent issues, this got aggregated and merged via the integrate branch: 67af063f372fe3764014069f0a765684786edd5c

**Pull request closed without merging**